### DISCUSSION TITLE
Play Jingle over Live Streams

### DISCUSSION THREAD

**Discussion by funkeeflow** [AzuraCast/AzuraCast#4837] [Q&A]
Hi 👋🏻 to the community!

Ahead of my question i want like to say, that this such a great project! While i found AzuraCast to be a perfect match to spin up a temporarily radio station, i try to figure my way around. So fare the experience has been flawless. Adding my own frontend using the api, setting up three parallel stations, checking out AutoDJs und so forth.

As we would like to host a virtual party during Christmas time where a bunch of dj will stream there live sets in a decentralized fashion through AzureCast to the listeners, giving them the ability to switch between stations( e.g. rooms), this worked out great so far.

But i'm wondering, what the best way would be, to play jingles (program announcements each station (room)) in an automated (timed) way during the event? Setting up a jingle playlist per station was my first guess, all though during live stream, they wont play of course or interrupt the program. I think the magic might be in defining my own liquidsoap config.

I appreciate any hints or ideas how to get in the right direction here!
Thanks in advance & cheers from Germany.

**Comment by SpeedyCharly** [comment]
Why don't you simply have the dj's  play the jingles as part of their live stream? That's how we do it at our station... 

We're using SAM Broadcaster; so we're using that largely useless Sound FX module by inputting the jingles into it and clicking them into the stream as needed.
![Image1](https://user-images.githubusercontent.com/66107794/145146559-4679d556-8078-47be-8dc2-f4b6a0830e5f.jpg)
